papers/valentina_staneva/myst.yml

```
@@ -0,0 +1,52 @@
+version: 1
+project:
+  # Update this to match `scipy-2024-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2024-valentina_staneva
+  # Ensure your title is the same as in your `main.md`
+  title: echodataflow: Configurable, Reproducible, and Scalable Fisheries Acoustics Workflows with Prefect
```

For the YAML, this needs quotes:

```suggestion
  title: "echodataflow: Configurable, Reproducible, and Scalable Fisheries Acoustics Workflows with Prefect"
```